Source/common/SNTConfigurator.m

```diff
@@ -1,4 +1,4 @@
-/// Copyright 2015 Google Inc. All rights reserved.
+/// Copyright 2021 Google Inc. All rights reserved.
```

Comment:
FYI: You don't need to update the copyright years - they should be correct when added and then left alone.

Reply:
Good to know.  It's an old habit from other jobs. I'll stop going forward.